mozci/console/commands/push.py

```diff
@@ -365,6 +373,19 @@ def handle(self) -> None:
                 f"<info>Written stats for {len(self.pushes)} pushes in {output}</info>"
             )
 
+        if self.option("detailed-classifications"):
+            self.line("\n")
+            for push in self.pushes:
+                if self.regressions.get(push) and (
+                    self.regressions[push]["real"]
+                    or self.regressions[push]["intermittent"]
+                ):
+                    self.line(
+                        f"<comment>Printing detailed classifications comparison for push {push.branch}/{push.rev}</comment>"
+                    )
+                    self.log_details(push, "real", {"fixed by commit"})
+                    self.log_details(push, "intermittent", {"intermittent"})
```

Comment:
We are checking that the results we marked as "real" failures are classified as "fixed by commit" and that those we marked as "intermittent" are classified as "intermittent".
@ahal @jmaher do you think we are missing something?

Reply:
I cannot think of any other situation where this could be a problem.  I have other thoughts about if we disable a test, is it a real failure or just a flaky/bad test that we turned on, but that is not relevant to this.  

An edge case might be infrastructure related, those jobs are often "blue" on treeherder or if not, they are marked as "intermittent".